modules/RTC/RTCUtils.js

```diff
@@ -377,7 +377,9 @@ class RTCUtils extends Listenable {
             ? { optional: [
                 { googScreencastMinBitrate: 100 },
                 { googCpuOveruseDetection: true }
-            ] }
+            ] + this.options.enableDSCP ? [
+                    { googDscp: true }
+                ] : [] }
```

Comment:
I don't think this syntax is correct, I tested it and it throws an error when enableDSCP is missing ...

Reply:
D'oh! Should be fixed now, and changed it to be handled the same way as the other options along with the log message when its value is set.